safe_control_gym/envs/gym_pybullet_drones/quadrotor.py

```diff
@@ -639,43 +639,49 @@ def _set_observation_space(self):
         self.phi_threshold_radians = 85 * math.pi / 180
         self.theta_threshold_radians = 85 * math.pi / 180
         self.psi_threshold_radians = 180 * math.pi / 180  # Do not bound yaw.
+        self.x_dot_threshold = 15
```

Comment:
same here, can we reorganize the thresholds to be in the same order as they appear in the obs? seems more organized.

Reply:
This reordering is done in the lates commit.